matplotcheck/base.py

```diff
@@ -909,19 +915,21 @@ def assert_xydata(
             xy_expected.sort_values(by=xcol),
         )
 
-        if tolerence > 0:
+        if tol_rel > 0 or tol_abs > 0:
             if xtime:
                 raise ValueError("tolerance must be 0 with datetime on x-axis")
             np.testing.assert_allclose(
                 xy_data["x"],
                 xy_expected[xcol],
-                rtol=tolerence,
+                rtol=tol_rel,
```

Comment:
@ryla5068 is the deal that the numpy allclose function has both parameters so you are exposing them in our method?